pyfaidx/__init__.py

```diff
@@ -460,7 +460,7 @@ def __repr__(self):
     def _index_as_string(self):
         """ Returns the string representation of the index as iterable """
         for k, v in self.index.items():
-            yield '\t'.join([k, str(v)])
+          yield '{k}\t{v.rlen:d}\t{v.offset:d}\t{v.lenc:d}\t{v.lenb:d}\n'.format(k=k, v=v)
```

Comment:
I like the idea of using the decimal formatter - this should avoid any issues with locale settings introducing strange separators. However I think I'll modify the `IndexRecord.__str__` method to accomplish this.

Reply:
I thought about doing that, but decided on this because it's not clear that other software isn't relying on the current behaviour, whereas _index_as_string is ostensibly a private implementation detail. It's your call, of course.

Reply:
That's a very respectful and thoughtful approach that I appreciate. Thanks again for your help with this.